packages/map/src/components/EditorPanel/components/EditorPanel.tsx

```diff
@@ -1288,6 +1288,9 @@ const EditorPanel = props => {
               })}
             </select>
           </label>
+            
```

Comment:
same comment as I had on the chart component. do these components do some filtering on their own somehow?

Reply:
See response above.